store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/operation/handler/impl/FederatedAddGraphHandlerTest.java

```diff
@@ -47,165 +49,175 @@
 
 public class FederatedAddGraphHandlerTest {
 
-    public static final String GAFFER_FEDERATEDSTORE_CUSTOM_PROPERTIES_AUTHS = "gaffer.federatedstore.customPropertiesAuths";
+    private static final String GAFFER_FEDERATEDSTORE_CUSTOM_PROPERTIES_AUTHS = "gaffer.federatedstore.customPropertiesAuths";
+    private static final String FEDERATEDSTORE_GRAPH_ID = "federatedStore";
+    private static final String EXPECTED_GRAPH_ID = "testGraphID";
+    private static final String EXPECTED_GRAPH_ID_2 = "testGraphID2";
+    private static final String CACHE_SERVICE_CLASS_STRING = "uk.gov.gchq.gaffer.cache.impl.HashMapCacheService";
+    private static final String FEDERATEDSTORE_CLASS_STRING = "uk.gov.gchq.gaffer.federatedstore.FederatedStore";
+    private static final String TEST_USER_ID = "testUser";
+    private FederatedStore store;
+    private StoreProperties federatedStoreProperties;
+
+    @Before
+    public void setUp() {
+        this.store = new FederatedStore();
+        federatedStoreProperties = new StoreProperties();
+        federatedStoreProperties.set(StoreProperties.STORE_CLASS, FEDERATEDSTORE_CLASS_STRING);
+        federatedStoreProperties.set(CacheProperties.CACHE_SERVICE_CLASS, CACHE_SERVICE_CLASS_STRING);
+    }
 
     @Test
     public void shouldAddGraph() throws Exception {
-
-        FederatedStore store = new FederatedStore();
-
         Schema expectedSchema = new Schema.Builder().build();
-        String expectedGraphId = "testGraphID";
 
-        StoreProperties storeProperties = new StoreProperties();
-        storeProperties.set("gaffer.store.class", "uk.gov.gchq.gaffer.federatedstore.FederatedStore");
+        StoreProperties graphStoreProperties = new StoreProperties();
+        graphStoreProperties.set(StoreProperties.STORE_CLASS, FEDERATEDSTORE_CLASS_STRING);
+        graphStoreProperties.set(CacheProperties.CACHE_SERVICE_CLASS, CACHE_SERVICE_CLASS_STRING);
```

Comment:
This graph probably doesn't need to be a FedStore this is an error carried over from before this ticket.